javascript/Spread_operator.md

```diff
@@ -115,79 +115,22 @@ The **syntax** of slice method is `array.slice(start, end)`.
 
 ### Example:
 #### Add to the End of an Array Using push()
-The push() method will add an element to the end of an array, while its twin function, the pop() method, will remove an element from the end of the array.
 
-To add an element to the end of an array using push(), you would do this:
 ```
  var list = ["foo", "bar"];
   list.push("baz");
   ["foo", "bar", "baz"] //result
 ```
-You can also add multiple elements to an array using push(), as shown below:
-```
-var list = ["foo", "bar"];
-list.push("baz", "qux", "etcetera");
-["foo", "bar", "baz", "qux", "etcetera"] // result
-```
-
-If you need to add an element or multiple elements to the end of an array, the push() method will almost always be your simplest and quickest option.
-
 #### Add to the Beginning of an Array Using unshift()
-The unshift() method will add an element to the beginning of an array, while its twin function, shift(), will remove one element from the beginning of the array.
-
-To add an element to the beginning of an array using unshift() try this:
 ```
 var list = ["foo", "bar"];
 list.unshift("baz");
 ["baz", "foo", "bar"] // result
```

Comment:
Remove these too.

Reply:
I have to remove just this two example or anything else?

Reply:
https://github.com/Techtonica/curriculum/pull/1345#discussion_r428028465